integration-tests/test_hub.py

```diff
@@ -45,10 +49,9 @@ def test_hub_up():
     hub_url = 'http://localhost'
     username = secrets.token_hex(8)
 
-    tljh_config_path = [sys.executable, '-m', 'tljh.config']
 
-    assert 0 == await (await asyncio.create_subprocess_exec(*tljh_config_path, 'add-item', 'users.admin', username)).wait()
```

Comment:
might be useful to define:

```python
async def check_call(command):
    """Call a subprocess and wait for it asynchronously

    Returns the exit code of the process
    """
    process = await asyncio.create_subprocess_exec(*command)
    return (await process.wait())

def tljh_config(*args):
    """Call tljh-config with arguments

    returns the exitcode of the process
    """
    return check_call(TLJH_CONFIG_PATH + list(args))

```

to be re-used, to avoid having to re-type the `await (await...` lines so often

Reply:
This code could also be simplified by calling the blocking `subprocess.check_call`, since there's no benefit to being async here, as far as I can tell.

Reply:
Is it ok if I punt this to a future PR? I'd like to keep using the async variants, since these tests need to be async anyway (for the hubtraf calls)

Reply:
Yes, absolutely! I just noticed a lot of duplicate code that could be simplified. async subprocess calls are very rarely beneficial and a lot more verbose.